### PR TITLE
[v5] Remove `actor.onDone(...)`

### PR DESCRIPTION
**Pull request by davidkpiano**
This PR removes `actor.onDone(...)` in favor of `actor.subscribe({ complete: () => { ... })`

### CODE REVIEW

File: packages/core/src/interpreter.ts [hunk 1]
```diff
@@ -99,6 +99,7 @@ export class Interpreter<
   private delayedEventsMap: Record<string, unknown> = {};
 
   private observers: Set<Observer<SnapshotFrom<TBehavior>>> = new Set();
+  private stopListeners: Set<() => void> = new Set();
```
Comment (Andarist) [inline packages/core/src/interpreter.ts:102]: was adding this intentional? currently, there is no way to add anything to this set

File: .vscode/launch.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
         "${file}",
         "--config",
         "jest.config.js",
-        "--no-cache",
-        "--testTimeout=2147483647"
+        "--no-cache"
+        // "--testTimeout=2147483647"
```
Comment (Andarist) [inline .vscode/launch.json:14]: this has to be reverted, which i guess just means that u need to merge `next` to this branch

**Review by Andarist — COMMENTED** [review]
this PR needs a changeset

File: packages/core/src/interpreter.ts (hunk 1 above)
Comment (davidkpiano) [reply packages/core/src/interpreter.ts:102]: It was not IIRC

**Review by Andarist — APPROVED** [review]

**Pull request merged**